Entities/Items/Lantern/Lantern.as

```diff
@@ -16,7 +16,7 @@ void onInit(CBlob@ this)
 	this.Tag("ignore fall");
 
 	this.getCurrentScript().runFlags |= Script::tick_inwater;
-	this.getCurrentScript().tickFrequency = 24;
+	this.getCurrentScript().tickFrequency = 1;
```

Comment:
You can remove this line since this is the default value.

Reply:
On second thought, it might be best to revert this back to 24. Lanterns are very cheap to make from the builder shop so making it tick so often could make it easier/quicker to lag out the server if a player spams tonnes of them. Plus, it's not really that important for it to be extinguished immediately upon contact with water.

Reply:
I tried lagging in offline and with both tick Frequency 1 and 24, it needs about 700 lanterns.
Compare to chicken, which needs about 280 of them to start noticing lag.
So the tick Frequency doesn't seem to make a difference. The blob itself being there is much more a drain on performance, than the function each one executes (which only consists of checking two booleans).
Chicken is much more a performance issue.

I think it is unlikely someone will actually reach 700 lanterns in CTF because they are destroying themselves due to overlapping with one another. And there is not enough space on the map to place 700 ones apart from each other.

This is why I created [ItemLimits.as](https://github.com/transhumandesign/kag-base/pull/1271). Assuming it does become a problem, I could change it to make CTF have its own item limits to cap things like Lantern and Bucket. The default would be infinite.